### PR TITLE
PassThroughEncoder capnproto implementation

### LINKED ISSUE #1992: Add capnp serialization to PassThroughEncoder
See #1715

### PR DESCRIPTION
**Pull request by oxtopus**
Builds off of #2001 and fixes #1992

cc @scottpurdy

### CODE REVIEW

**Comment by rhyolight** [comment]
Restarted build...

File: nupic/encoders/pass_through_encoder.capnp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@0xa9b5ded538ae8ac4;
```
Comment (scottpurdy) [inline nupic/encoders/pass_through_encoder.capnp]: `pass_through.capnp`?

**Comment by rhyolight** [comment]
WTF is wrong with Travis-CI?

**Comment by oxtopus** [comment]
I don't know, but the failures are holding me back from merging with a good
conscience.

**Comment by rhyolight** [comment]
I contacted travis support.

**Pull request closed without merging**